tests/model/DBStringTest.php

```diff
@@ -26,6 +27,18 @@ public function testForTemplate() {
 		);
 	}
 
+	public function testDefault() {
+		/** @var DBString $dbField */
+		$dbField = Object::create_from_string(
+			"StringFieldTest_MyStringField(['default' => 'Here is my default text')",
```

Comment:
Doesn't seem to break the test, but missing a closing bracket for the array `"StringFieldTest_MyStringField(['default' => 'Here is my default text'])"`